buildh/geometry.py

```diff
@@ -95,7 +94,7 @@ def calc_rotation_matrix(quaternion):
         The rotation matrix.
     """
     # Initialize rotation matrix.
-    matrix = np.zeros([3, 3])
+    matrix = np.zeros((3, 3))
     # Get quaternion elements.
```

Comment:
I guess here you use a tuble because it's faster than a list when using literals, right?

Reply:
No, in fact, np.zeros takes as argument a shape, which is usually a tuple.
Plus, otherwise, it fails when using numba decorator

Reply:
In principle, np.zeros() supports a list or a tuple for indicating the array shape, there's no problem. But I didn't know it did not with numba!